task_26/toma.lobanova/src/main/java/com/hillel/tasks/converting/TxtToXml.java

```diff
@@ -0,0 +1,75 @@
+package com.hillel.tasks.converting;
+
+import com.hillel.tasks.student.Student;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStreamWriter;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TxtToXml {
+  private List<Student> students;
+
+  public TxtToXml() {
+    students = new ArrayList<>();
+  }
+
+  public void readStudents() throws IOException {
+    InputStream input = getClass().getResourceAsStream("students.txt");
+    try (BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(input, StandardCharsets.UTF_8))) {
+      String line;
+      List<String> studentFields = new ArrayList<>();
+      do {
+        line = readOneStudentFields(bufferedReader, studentFields);
+        addNewStudentToList(studentFields);
+      }
+      while (line != null);
+    }
+  }
+
+  private String readOneStudentFields(BufferedReader bufferedReader, List<String> studentFields) throws IOException {
+    String line;
+    studentFields.clear();
+    while ((line = bufferedReader.readLine()) != null && !line.equals("")) {
```

Comment:
line.equals("") can be replaced by line.isEmpty() method, it does the same but looks a bit nicer